pepper-apis/docs/specification/deploy/app.yaml

```diff
@@ -0,0 +1,18 @@
+service: pepper-api-spec
+runtime: python37
+
+# This setting will not matter except to prevent default automatic-scaling which prevents disabling
+# earlier versions of this service
+basic_scaling:
+  max_instances: 1
+
+handlers:
+  - url: /spec/(.*\.(yaml|json))
+    static_files: spec/\1
+    upload: spec/(.*\.(yaml|json))
+    secure: always
+
+  - url: /(.*)
+    static_files: index.html
+    upload: index.html
+    secure: always
```

Comment:
This is a purely static file site. Stuff to with scaling and python there just to keep GAE happy; no instances will be created.